csrc/host_ir/executor.cpp

```diff
@@ -281,6 +281,25 @@ void HostIrExecutor::handle(EndCoalescing* end_coalescing) {
   works_[end_coalescing] = backend->endCoalescing();
 }
 
+void HostIrExecutor::handle(MatmulOp* matmul) {
+  TensorView* a = matmul->inA();
+  TensorView* b = matmul->inB();
+  TensorView* out = matmul->out();
+  NVF_ERROR(
+      expr_evaluator_.isKnown(a) && expr_evaluator_.isKnown(b),
+      "Inputs of the matmul ",
+      matmul->toString(),
+      "must be precomputed before being retrieved");
+  if (expr_evaluator_.isKnown(out)) {
```

Comment:
Can you verify both branches are tested? I suspect the test you added to test_host_irs only covers the if-else branch.

Reply:
this comment applies to https://github.com/NVIDIA/Fuser/pull/3171
ported to https://github.com/NVIDIA/Fuser/pull/3171/files#r1800452454